### ISSUE TITLE
Fulcrum Jekyll Links Have Insufficient Color Contrast

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#608]
LINKS: Element has insufficient color contrast of 2.52 (foreground color: #00afec, background color: #ffffff, font size: 17.3pt, font weight: normal)

Example: 
`body > .page-content > div:nth-of-type(4) > .section.case-study > div:nth-of-type(2) > .col.s12.m8.project-info > h3 > a`
`<a href="https://www.fulcrum.org/concern/monographs/5999n3367">Canoes: A Natural History in North America</a>`

- [ ] 1.4.3 Contrast (Minimum): The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following: (Level AA)
Large Text: Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental: Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes: Text that is part of a logo or brand name has no minimum contrast requirement.
